Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -17,17 +17,13 @@
 ////////////////////////////////////////////////////////////////////////////
 
 #import "RLMSyncTestCase.h"
+#import "RLMTestUtils.h"
 #import "RLMSyncSessionRefreshHandle+ObjectServerTests.h"
 #import "RLMSyncUser+ObjectServerTests.h"
 #import "RLMSyncUtil_Private.h"
 #import "RLMRealmConfiguration_Private.h"
 #import "RLMRealmUtil.hpp"
 
-#define ACCOUNT_NAME() NSStringFromSelector(_cmd)
```

Comment:
Thomas expressed the opinion that the macro was needlessly obfuscating in his earlier review. Happy to change it back if you think the readability benefit makes it worth keeping.